[2024-04-21T19:54:48.519+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: workshop2_dags.extract_csv manual__2024-04-21T18:58:03.284812+00:00 [queued]>
[2024-04-21T19:54:48.533+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: workshop2_dags.extract_csv manual__2024-04-21T18:58:03.284812+00:00 [queued]>
[2024-04-21T19:54:48.534+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 3
[2024-04-21T19:54:48.579+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_csv> on 2024-04-21 18:58:03.284812+00:00
[2024-04-21T19:54:48.591+0000] {standard_task_runner.py:60} INFO - Started process 91 to run task
[2024-04-21T19:54:48.599+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'workshop2_dags', 'extract_csv', 'manual__2024-04-21T18:58:03.284812+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/Airflow/dags_etl.py', '--cfg-path', '/tmp/tmp5p8kky_m']
[2024-04-21T19:54:48.601+0000] {standard_task_runner.py:88} INFO - Job 31: Subtask extract_csv
[2024-04-21T19:54:59.062+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 31 for task extract_csv ((mysql.connector.errors.DatabaseError) 2005 (HY000): Unknown MySQL server host 'mysql' (-3)
(Background on this error at: https://sqlalche.me/e/14/4xp6); 91)
